homeassistant/components/wemo/strings.json

```diff
@@ -10,6 +10,22 @@
       "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]"
     }
   },
+  "options": {
+    "step": {
+      "init": {
+        "data": {
+          "enable_subscription": "Subscribe to device local push updates",
+          "enable_long_press": "Register for device long-press events",
+          "polling_interval_seconds": "Seconds to wait between polling the device"
```

Comment:
We generally don't allow polling / scan interval as option. Every integration that supports config entries allows the user to turn off automatic polling and then the user can automate the update interval as needed with the entity service `homeassistant.update_entity`.

Please remove this option.

Reply:
This is something else/special where a very specific edge case is handled. This integration is push based and does not require polling at all, just a few misbehaving devices as it seems.

Reply:
I don't see anything special that should allow this option. The integration is using coordinators that support automating the poll with the entity service call.

If there's some reason that can't be used that needs to be clarified and approved before we can accept it.

Reply:
Its a bit unfortunate that we did not provide that comment back in 2021 when this PR was first submitted then. 
I'm not a fan of having the user manually poll the device (as that is very user unfriendly imo) but considered this is an edge case I guess that is fine and can be explained in the docs. If you are 100`5 certain that we can replicate this behavior using builtin features, that is even better but I think we should now help @esev implementing that.

Reply:
@esev sorry for the confusion. We have builtin logic to poll devices so we should use that if push updates are disabled for devices. A user can even disable that automatic poll and then poll manually to get custom intervals. So we need to address that in a next PR within this beta period, otherwise this PR will be reverted. Sorry for the unclarity but @MartinHjelmare is right that we can do this using default options.

**The next PR should address this:**

1) Still have an option to disable push updates in the integration's config options.
2) Set "should_poll" to True within the entities.
3) Remove the option to set the poll interval from your options.
4) Users that want maximum control over the poll interval, van disable automatic polling and use the 'homeassistant.update_entity' service with their own interval.

Just ping me for the quick review if your PR is open. 
Thanks!

Reply:
2. is not needed as this integration relies on coordinators.

1. Is just what this PR does, so no change is needed in the next PR for that.

Reply:
Thanks for the info! I'm testing a PR now. Will send it in a few hours.

Reply:
I added a fix in #95468